src/Client/components/ques/written-ques.tsx

```
@@ -1,200 +1,331 @@
+                                                question.id,
+                                                e.target.checked,
+                                                e.target.checked
+                                                    ? false
+                                                    : question.isLongAnswer
+                                            )
+                                        }
+                                        isDisabled={question.isLongAnswer}
+                                    >
+                                        Short Answer
+                                    </Checkbox>
+                                </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)